packages/SwingSet/src/kernel/deviceManager.js

```diff
@@ -78,7 +78,7 @@ export default function makeDeviceManager(
     insistVatType('object', targetSlot);
     insistCapData(args);
     const target = mapDeviceSlotToKernelSlot(targetSlot);
-    assert(target, `unable to find target`);
```

Comment:
I thought we backed off on that rule. I was leaving backquotes so that folks could easily add actual data fields.